src/Base/Exception.C

```diff
@@ -96,14 +92,14 @@ Exception::saveTrace() noexcept
 //!   Requires stdio.h, execinfo.h.
 // *****************************************************************************
 {
+#ifndef HOST_OS_ALPINE
```

Comment:
I would really like to see a comment why ALPINE doesn't have it....

Reply:
It does, but segfaults on calling `backtrace()`, and I have not clue how to fix it. I tried playing with the buffer length it takes its 2nd argument, as well as putting the buffer in global scope, making it static, etc. I just cannot fix it. So on alpine we cannot call it.

Reply:
Also, https://www.freshports.org/devel/libexecinfo says it's quick'n'dirty, so it is probably broken.

Reply:
Still a comment about alpine OS would be nice

Reply:
I know the commit message has it, but I mean right here in the code.

Reply:
Ah, okay, I see. I will put in a comment there then. Thx.